firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionDetailsViewController.swift

```diff
@@ -0,0 +1,280 @@
+            connectionView.trailingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.trailingAnchor,
+                constant: -TPMenuUX.UX.horizontalMargin
+            ),
+            connectionView.heightAnchor.constraint(greaterThanOrEqualToConstant: TPMenuUX.UX.baseCellHeight),
+        ]
+
+        constraints.append(contentsOf: connectionViewContraints)
+    }
+
+    // MARK: Verified By View Setup
+    private func setupVerifiedByView() {
+        baseView.addSubview(verifiedByView)
+
+        let verifiedByViewContraints = [
+            verifiedByView.leadingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.leadingAnchor,
+                constant: TPMenuUX.UX.horizontalMargin
+            ),
+            verifiedByView.topAnchor.constraint(equalTo: connectionView.bottomAnchor),
+            verifiedByView.bottomAnchor.constraint(equalTo: baseView.bottomAnchor,
+                                                   constant: -TPMenuUX.UX.TrackingDetails.bottomDistance),
+            verifiedByView.trailingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.trailingAnchor,
+                constant: -TPMenuUX.UX.horizontalMargin
+            ),
+            verifiedByView.heightAnchor.constraint(greaterThanOrEqualToConstant: TPMenuUX.UX.baseCellHeight)
+        ]
+
+        constraints.append(contentsOf: verifiedByViewContraints)
+    }
+
+    // MARK: See Certificates View Setup
+    private func setupSeeCertificatesView() {
+        let certificatesButtonViewModel = LinkButtonViewModel(
+            title: model.viewCertificatesButtonTitle,
+            a11yIdentifier: model.viewCertButtonA11Id,
+            font: FXFontStyles.Regular.footnote.scaledFont()
+        )
+        viewCertificatesButton.configure(viewModel: certificatesButtonViewModel)
+        baseView.addSubview(viewCertificatesButton)
+
+        let viewCertificatesConstraints = [
+            viewCertificatesButton.leadingAnchor.constraint(
+                equalTo: view.leadingAnchor,
+                constant: TPMenuUX.UX.horizontalMargin
+            ),
+            viewCertificatesButton.trailingAnchor.constraint(
+                equalTo: view.trailingAnchor,
+                constant: -TPMenuUX.UX.horizontalMargin
+            ),
+            viewCertificatesButton.topAnchor.constraint(
+                equalTo: verifiedByView.bottomAnchor,
+                constant: TPMenuUX.UX.TrackingDetails.viewCertButtonTopDistance
+            ),
+        ]
+
+        constraints.append(contentsOf: viewCertificatesConstraints)
+    }
+
+    // MARK: Accessibility
+    private func setupAccessibilityIdentifiers() {
+        headerView.accessibilityIdentifier = model.headerViewA11yId
+        view.accessibilityIdentifier = model.detailsViewA11yId
+        baseView.accessibilityIdentifier = model.containerViewA11Id
+        connectionView.accessibilityIdentifier = model.connectionViewA11yId
```

Comment:
why not set the a11y within each own view? maybe add a configure for each view to set its own properties in the init?

Reply:
yeah, I wanted to see how it looks if I handle it separately for the whole controller but in the end there were too many subviews for it to be pretty

Reply:
okay, a comment here to clarify would be helpful